internal/acceptance/mws_networks_test.go

```diff
@@ -22,3 +22,25 @@ func TestMwsAccNetworks(t *testing.T) {
 		}`,
 	})
 }
+
+func TestMwsAccGcpPscNetworks(t *testing.T) {
+	accountLevel(t, step{
+		Template: `
+		resource "databricks_mws_networks" "my_network" {
+			account_id   = "{env.DATABRICKS_ACCOUNT_ID}"
+			network_name = "network-test-{var.RANDOM}"
+			gcp_network_info {
+			  network_project_id = "{env.GOOGLE_PROJECT}"
+			  vpc_id = "env.VPC_NETWORK_ID"
+			  subnet_id = "env.SUBNET_ID"
+			  subnet_region = "{env.GOOGLE_REGION}"
+			  pod_ip_range_name = "pod"
+			  service_ip_range_name = "svc"
+            }
+            vpc_endpoints {
+        	  rest_api = ["{env.REST_API_VPCE_ID}"]
```

Comment:
These environment variables don't exist in our test infrastructure. I'll show you how to add those on Monday